querybook/webapp/components/DataDocScheduleList/cronHelper.tsx

```diff
@@ -0,0 +1,57 @@
+import { getRecurrenceLocalTimeString, WEEKDAYS, MONTHS } from 'lib/utils/cron';
+
+function formatMultipleItems(data, executor, initialParam) {
+    return data.slice(1).reduce((acc, item, idx, arr) => {
+        if (idx === arr.length - 1) {
+            return `${acc} and ${executor(item)}`;
+        } else {
+            return `${acc}, ${executor(item)}`;
+        }
+    }, initialParam);
+}
+
+export const hourly = ({ minute }) =>
```

Comment:
this is really nice!, a few things: 
1. can you add typing to execution? i don't think `execution` is a good name for a cron recurrence
2. can you export a single function to do it:
```
export const cronFormatter(recurrence: IRecurrence) {
    const recurrenceType = recurrence.type;
    if (recurrenceType === 'daily') {
        ...
}